nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -700,6 +814,17 @@ bool RegulatedPurePursuitController::transformPose(
   return false;
 }
 
+double RegulatedPurePursuitController::angleNormalize(double angle)
```

Comment:
`ros/angles` has these types of functions implemented already that can be used, remove from here and use the standard implementations

Reply:
Thanks, very good suggestion.